config/resources.txt

```diff
@@ -37,3 +37,9 @@ ASSET_CDN_WEBROOT data/asset-store/
 # if you want to test this locally, you simpily run the `localhost-asset-webroot-server.py` python3 script to host assets stored in `data/asset-store/` via http://localhost:7123/
 ASSET_CDN_URL http://localhost:7123/
 
+## Assets can opt-in to caching their results into `tmp`.
+## This is important, as preferences assets take upwards of 30 seconds (without sleeps) to collect.
+## The cache is assumed to be cleared by TGS recompiling, which deletes `tmp`.
+## This should be disabled (through `CACHE_ASSETS 0`) on development
+## This should be enalbed by 1 from server side
+CACHE_ASSETS 0
```

Comment:
```suggestion
########################
# Asset Caching Config #
########################
## Assets can opt-in to caching their results into `tmp`.
## This is important, as preferences assets take upwards of 30 seconds (without sleeps) to collect.
## The cache is assumed to be cleared by TGS recompiling, which deletes `tmp`.
## This should be uncommented on production
#CACHE_ASSETS
```

You don't actually need the whole `0` for disable and `1` for enabled now. You just comment it out to disable it and uncomment it to enable it.